flake.nix

```diff
@@ -110,6 +110,15 @@
           type = "app";
           program = "${self.packages.${system}.default}/bin/llama-server";
         };
+        apps.llama-server-openai-proxy = {
```

Comment:
I was going to mention this in https://github.com/ggerganov/llama.cpp/pull/4605, but we wouldn't have had to manually wrap these in `apps` if we instead exposed them (in the overlay and) in packages/legacyPackages, and if we ensured that `meta.mainProgram`s are set correctly (with `writeShellScriptBin` they are)

Reply:
You could now add this to `scope.nix`: https://github.com/ggerganov/llama.cpp/blob/4e1d2510e445bb98c221c7b1d2ded20ffae60cab/.devops/nix/scope.nix#L7-L12

This way the attribute would show up as `llamaPackages.llama-server-openai-proxy` in nixpkgs with the flake's `overlays.default` applied, as well as in the flake's `legacyPackages.${system}.llamaPackages`.